mail-client/bower/bower-0.10.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit multiprocessing
+
+DESCRIPTION="A curses terminal client for the Notmuch email system"
+HOMEPAGE="https://github.com/wangp/bower"
+SRC_URI="https://github.com/wangp/bower/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+MY_DEPEND="app-crypt/gpgme:=
```

Comment:
I did not come up with `COMMON_DEPEND` out of the blue, we follow best practice:
```
gentoo.git (master) $ grep -R COMMON_DEPEND= | wc -l
1131
gentoo.git (master) $ grep -R MY_DEPEND= | wc -l
0
```

Reply:
That was actually not deliberate on my end. Prefixing variables with `MY_` has become sort of a reflex.